demos/browser/app/meetingReadinessChecker/meetingReadinessChecker.ts

```diff
@@ -77,6 +80,8 @@ export class DemoMeetingApp {
     this.initParameters();
     this.setMediaRegion();
     this.switchToFlow('flow-authenticate');
+    const button = document.getElementById("authenticate") as HTMLButtonElement;
+    button.disabled = false;
   }
 
   switchToFlow(flow: string): void {
```

Comment:
We do not have flow in this demo (unless you add more), could we rename this function to something better?

Reply:
We have 3 now. 'flow-authenticate', 'flow-failed-meeting' & 'flow-readinesstest'. So I think so it makes sense to keep it like this.